examples/incredible-dot-product/src/config/dot-operator.toml

```
@@ -9,9 +9,7 @@ aggregator_ip_port = "127.0.0.1:8080"
 
 
 [registration]
-operator_pvt_key = "0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80"
-ecdsa_keystore_path = ""
-ecdsa_keystore_password = ""
+signer = { PrivateKey = "0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80" }
```

Does this work like:

```suggestion
signer.private_key = "0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80"
```

---

Nop 
```bash
thread 'main' panicked at examples/incredible-dot-product/src/bin/operator.rs:26:80:
called `Result::unwrap()` on an `Err` value: Could not parse config file: TOML parse error at line 12, column 8
   |
12 | signer.private_key = "0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80"
   |        ^^^^^^^^^^^
unknown variant `private_key`, expected `PrivateKey` or `Keystore`
```

But it works with this:

`signer.PrivateKey = "0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80"`